CONTRIBUTING.md

```diff
@@ -40,15 +40,38 @@ with `gn format`.
 
 ## Testing
 
-The testing policy for contributing to the flutter engine can be found at the [Tree Hygiene Wiki][tree_hygiene_wiki]. The summary is that all PR's to the engine should be tested or have an explicit test exemption.
+The testing policy for contributing to the flutter engine can be found at the
+[Tree Hygiene Wiki][tree_hygiene_wiki]. The summary is that all PR's to the
+engine should be tested or have an explicit test exemption.
 
-Because the engine targets multiple platforms the testing infrastructure is fairly complicated. Here are some more resources to help guide writing tests:
+Because the engine targets multiple platforms the testing infrastructure is
+fairly complicated. Here are some more resources to help guide writing tests:
 
-- [Testing the engine wiki][testing_the_engine_wiki] - A guide on writing tests for the engine including an overview of the different tests and the different technologies the engine uses.
-- [//testing](./testing) - This is where the `run_tests.py` script is located. All tests will have the ability to be executed with `run_tests.py`.
-- [//ci/builders](./ci/builders) - The JSON files that determine how tests are executed on CI.
+- [Testing the engine wiki][testing_the_engine_wiki] - A guide on writing tests
+  for the engine including an overview of the different tests and the different
+  technologies the engine uses.
+- [`//testing`](./testing) - This is where the `run_tests.py` script is located.
+  All tests will have the ability to be executed with `run_tests.py`.
+- [`//ci/builders`](./ci/builders) - The JSON files that determine how tests are
+  executed on CI.
 
-Tests will be executed on CI, but some tests will be executed before PR's can be merged (presubmit) and others after they have been merged (postsubmit). Ideally everything would be presubmit but tests that take up more resources are executed in postsubmit.
+Tests will be executed on CI, but some tests will be executed before PR's can be
+merged (presubmit) and others after they have been merged (postsubmit). Ideally
+everything would be presubmit but tests that take up more resources are executed
+in postsubmit.
+
+### Skia Gold
+
+The Flutter engine uses [Skia Gold][] for image comparison tests which fail if:
```

Comment:
This is a broken link methinks.

Reply:
My understand is the implicit default is lowercased, but I can make it explicit.

Reply:
could be, it's hard to verify in githubs editor =T  thanks